<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoodsDetailDialog</class>
 <widget class="QDialog" name="GoodsDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoodsDetailDialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 0.7); 
    border-radius: 10px;                        
    border: 1px solid rgba(0, 0, 0, 0.2);       
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ClickableLabel" name="goodsPicture">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resource.qrc">:/res/default.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="goodsID1">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
           </property>
           <property name="text">
            <string>ID：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goodsIDEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="goodsName1">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
           </property>
           <property name="text">
            <string>名字：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goodsNameEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="goodsPrice1">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
           </property>
           <property name="text">
            <string>价格：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goodsPriceEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="goodsCategory1">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
           </property>
           <property name="text">
            <string>类别：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goodsCategoryEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="goodsCount1">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
           </property>
           <property name="text">
            <string>库存：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goodsCountEdit">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>NULL</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="goodsDescription">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
           </property>
           <property name="text">
            <string>描述：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="goodsTextEdit">
           <property name="styleSheet">
            <string notr="true">QTextEdit {
    background-color: rgba(255, 255, 255, 1); 
    border-radius: 10px;                        
    border: none;       
}</string>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;null&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget {
    background-color: rgba(255, 255, 255, 0.0); 
    border-radius: 10px;                        
    border: none;       
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="acceptBtn">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton 
{
    background-color: #2196F3;  
    color: white;               
    border: none;              
    padding: 10px 20px;        
    border-radius: 15px;       
    font-size: 14px;           
}

QPushButton:hover
{
    background-color: #1976D2;  
}

QPushButton:pressed 
{
    background-color: #0D47A1;
    color: #E0E0E0;            
}</string>
           </property>
           <property name="text">
            <string>确认修改</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelBtn">
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton 
{
    background-color: #2196F3;  
    color: white;               
    border: none;              
    padding: 10px 20px;        
    border-radius: 15px;       
    font-size: 14px;           
}

QPushButton:hover
{
    background-color: #1976D2;  
}

QPushButton:pressed 
{
    background-color: #0D47A1;
    color: #E0E0E0;            
}</string>
           </property>
           <property name="text">
            <string>取消修改</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
